editor.planx.uk/src/@planx/components/ui.tsx

```diff
@@ -109,46 +110,44 @@ export const MoreInformation = ({
   return (
     <ModalSection>
       <ModalSectionContent title="More Information" Icon={InfoOutlined}>
-        <InputGroup label="Why it matters">
-          <InputRow>
+        <InputGroup flowSpacing>
+          <InputLabel label="Why it matters">
```

Comment:
Do we want to remove the `placeholder` values now if we're adding labels here?

Reply:
Good shout, the label + placeholder combo is already in place but it makes sense to have one or the other.